mystory/tests.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from mystory.models import Mystory
+
+
+TEMP_DATE = '2006-10-25 14:30:59'
+TEMP_STORY = "This is my story"
+
+
+# @pytest.mark.django_db()
+# def test_new_item():
+# item = Mystory(story_date=TEMP_DATE, story_text=TEMP_STORY)
+# item.save()
+
+# assert item.story_text == TEMP_STORY
+
+
+@pytest.fixture
+def astory():
+    return Mystory(story_date=TEMP_DATE, story_text=TEMP_STORY)
```

Comment:
Im not sure but i think you need to .save() the model
try that after expenses app updates with the migrations